tutorials/mongoose/connection-status.md

```diff
@@ -0,0 +1,9 @@
+['Mongoose has a readyState property in the Connection route of the API']('https://mongoosejs.com/docs/api/connection.html#connection_Connection-readyState') that is of type **property** and will return a number representing the current state of the connection, 0-4. These states are as follows:
```

Comment:
Why are there single quotes around 'Mongoose has a ...' and the `https://mongoosejs.com` url?

Reply:
my bad on the single quotes, I was looking at the timestamps.md to see how to link to something and thought I saw single quotes in those sections. Taking them out now.